misc/start_faux

```diff
@@ -130,5 +144,14 @@ else
     echo Runtime config $conf_file not found.
 fi
 
+cat <<EOF >> variables.debug
```

Comment:
what is this for?

Reply:
Removed the following statement as it has no other references and was primarily used for debugging in the faux device:

```
cat <<EOF >> variables.debug
bac0_port= $bac0_port
multicast_group= $multicast_group
duration_seconds= $duration_seconds
local_ip= $local_ip
broadcast_ip= $broadcast_ip
intf_name= $intf_name
conf_file= $conf_file
EOF
```